packages/pwa/app/utils/utils.js

```diff
@@ -157,6 +157,13 @@ export const capitalize = (text) => {
         .join(' ')
 }
 
+/**
+ * A util to return current url configuration
+ * @returns {object} - url object from the pwa-kit-config.json file
+ */
+ export const getConfig = () => console.log('TODO')
```

Comment:
I will need to change the import paths for all the file that requires the config, but i'd like to do it in another PR because that's going to make this PR hard to read.